src/nextcloud/talk/recording/Service.py

```diff
@@ -24,6 +25,8 @@
 
 import logging
 import os
+import sys
+import platform
```

Comment:
Both imports look like leftovers from previous experiments, as they are not used anywhere in the file :-)

Reply:
yes, most certainly they are :) I made the changes on the test server directly with a limited vim setup :D